stubs/2.7/csv.pyi

```diff
@@ -0,0 +1,103 @@
+# Stubs for csv (Python 2.7)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from abc import ABCMeta, abstractmethod
+from typing import Any, Dict, Iterable, List, Sequence, Union
+
+# Public interface of _csv.reader
+class Reader(Iterable[List[str]], metaclass=ABCMeta):
+    dialect = ...  # type: Dialect
+    line_num = ...  # type: int
+
+    @abstractmethod
+    def next(self) -> List[str]: ...
+
+_Row = Sequence[Union[str, int]]
+
+# Public interface of _csv.writer
+class Writer(metaclass=ABCMeta):
+    dialect = ...  # type: Dialect
+
+    @abstractmethod
+    def writerow(self, row: _Row) -> None: ...
+
+    @abstractmethod
+    def writerows(self, rows: List[_Row]) -> None: ...
```

Comment:
Actually, the documentation also says `writer.writerows(someiterable)` so I'll just use iterable.